fv_be/app/models/sites.py

```diff
@@ -164,3 +164,21 @@ class Meta:
 
     def __str__(self):
         return f"{self.key}: {self.is_enabled} ({self.site})"
+
```

Comment:
The idea here is to only store a SiteMenu if a site has customized it. Otherwise we can return a default value that is stored in the AppJson model.